diff --git a/lib/supports.ts b/lib/supports.ts
index 902b61a..3ee6d64 100644
--- a/lib/supports.ts
+++ b/lib/supports.ts
@@ -17,6 +17,7 @@ export const Supports = new (class {
 	}

 	isBrowserSupported(): boolean {
+		if(navigator.product === 'ReactNative') return true;
 		const browser = this.getBrowser();
 		const version = this.getVersion();

@@ -41,6 +42,7 @@ export const Supports = new (class {
 	}

 	isUnifiedPlanSupported(): boolean {
+		if(navigator.product === 'ReactNative') return true;
 		const browser = this.getBrowser();
 		const version = webRTCAdapter.browserDetails.version || 0;

